controllers/designateworker_controller.go

```diff
@@ -487,6 +487,25 @@ func (r *DesignateWorkerReconciler) reconcileNormal(ctx context.Context, instanc
 	// create hash over all the different input resources to identify if any those changed
 	// and a restart/recreate is required.
 	//
+
+	// If DesignateBindKeySecret exists, add its hash to status hash
+	rndcSecret := &corev1.Secret{}
```

Comment:
I'm wondering if it can be integrated directly into createHashOfInputHashes (called line 509) which does almost the same thing.
I guess you only have to do
`configMapVars[designate.DesignateBindKeySecret] = env.SetValue(secretHash)`

Reply:
I have just tried it and it works. That method indeed does almost the same thing. Thanks